### ISSUE TITLE
Module not found 'sklearn.semi_supervised.label_propagation'

### ISSUE THREAD

**Issue by dimaelzein** [EFord36/normalise#124]
- Normalised was successfully installed using pip
- normalise function not recognized in the code
- I tried "Import normalise" , received the following error: 'sklearn.semi_supervised.label_propagation'.  Installing this module :  Could not find a version that satisfies the requirement sklearn.semi_supervised.label_propagation (from versions: none).  Didn;t work even when sklearn downgraded to 0.20.3
- 
Python 3.9.4 , pip 21.1.1

**Comment by dophist** [comment]
same here

**Comment by damorosodaragona** [comment]
Same problem. 
Has anyone found a solution?

**Comment by Techseeker-404** [comment]
Faced similar problem.

**Comment by sylzys** [comment]
same here

**Comment by ntdat017** [comment]
Downgrade scikit-learn version to <= v0.22.1 to fix it.

**Comment by HanyinWang** [comment]
scikit-learn 0.23.1 works for me
